src/Config/MosaicConfig.schema.json

```diff
@@ -151,9 +151,7 @@
       "required": [
         "gatewayLibAddress",
         "merklePatriciaLibAddress",
-        "messageBusAddress",
-        "simpleTokenAddress",
-        "ostComposerAddress"
+        "messageBusAddress"
```

Comment:
why these addresses are removed?
they should be mandatory.

Reply:
Because mosaic libraries deploys  libraries and generates mosaic config with libraries. If these fields are made mandatory then generated mosaic config is invalid